server/server.js

```diff
@@ -115,17 +116,21 @@ app.patch(`${apiBaseUrl}/form/:id`,
     await validate(DeterminationSchema, req.body);
     const { id } = req.params;
     const formsCollection = dbClient.db.collection(collections.FORMS);
+    const currentDate = new Date().toISOString();
 
     await formsCollection.updateOne(
       { id }, // Query
       { // UpdateQuery
         $set: {
           notes: req.body.notes,
           determination: req.body.determination,
-          updatedAt: new Date().toISOString(),
+          updatedAt: currentDate,
         },
       },
     );
+
+    logger.verbose(`Form ${id} updated by "${req.user.id}" with determination "${req.body.determination}" and notes "${req.body.notes}"`, currentDate);
```

Comment:
This looks good but are we sure that `notes` will be PII free?

Reply:
Not sure. But it is a ticket requirement. Maybe @fresh-rachit could give a better answer here.

Reply:
There is a non zero chance that notes will contain PI and I’d like to keep all PI in the database instead of streaming to cloudwatch. So let’s record when a record was changed with the timestamp but not what was changed in the notes field. For example, if the notes field was changed 3 times, there are 3 logs in cloudwatch but the content of what was inserted/edited is not visible.

Reply:
Update looks good!